packages/gatsby-theme-docs/src/templates/release-notes-list.js

```diff
@@ -31,11 +32,13 @@ const ReleaseNotesListTemplate = (props) => (
             {props.data.allReleaseNotePage &&
               props.data.allReleaseNotePage.nodes &&
               props.data.allReleaseNotePage.nodes.map((releaseNote) => (
-                <LayoutReleaseNote
-                  key={releaseNote.slug}
-                  bodyRenderer="excerpt"
-                  {...releaseNote}
-                />
+                <LayoutReleaseNote key={releaseNote.slug} {...releaseNote}>
+                  <Markdown.TypographyPage>
+                    <section>
```

Comment:
@nkuehn should these elements have some more contextual info? Like ids, labels, ...